src/chat/Chat.js

```diff
@@ -9,12 +9,28 @@ import MessageList from '../message-list/MessageList';
 import NoMessages from '../message/NoMessages';
 import ComposeBox from '../compose/ComposeBox';
 
-export default class MainScreen extends React.Component {
+export default class Chat extends React.Component {
+
+  unsentMessageIds = [];
```

Comment:
We'll have to be careful with this state because it will could be retained when switching accounts. This is an existing problem though I think